V8 Migration - Added information about obsolete Data Types

HannesWinston opened the pull request:
Just a simple reminder to go through the Umbraco 7 site for obsolete  Data Type properties before continuing with the migration, as it will otherwise fail. Could save someone from a headache while migrating...

sofietoft commented:
This is a great addition @HannesWinston !

Would it make sense to also add a list of the Data Types that actually have an updated counterpart? Just as a quick little "checklist" or overview.

HannesWinston commented:
I think that would be good. The obsolete Data Types I encountered while migrating an old site to Umbraco 8 were:

- Media Picker
- Content Picker
- Multinode Tree Picker
- Nested Content
- Member Picker
- Folder Browser
- Related Links

I can add this list to the docs, but I don't know if those are all of the updated Data Types. You probably know this better than I do :)

sofietoft commented:
I can confirm that that is all of them 👏 

You're more than welcome to add the list yourself, if you want?
And perhaps add a small mention about how they're called `(Obsolete) Media Picker` etc.? 😁

HannesWinston commented:
Wonderful! I have added the list of properties and added a small mention about the name.

I couldn't find an example of a list within a warning box in the documentation, so I hope it works on the site. Otherwise, I can move the information out of the warning box to regular text.

sofietoft commented:
Awesome, thanks! 🎉 

I'm sure it'll work.
I can also see that this is the first time you're contributing to the Umbraco Documentation?
That's so cool!! #h5yr
Do you have an account on [Our Umbraco](https://our.umbraco.com/)? 
If you do, let me know your username, and I'll grant you the "C-Trib" badge 😎

HannesWinston commented:
Cool! Yes it is my first time contributing :)

Just created an account on our umbraco, linked to this github account HannesWinston.
Thanks!
